applications/data_analysis/spectral_analysis.ipynb

```diff
@@ -0,0 +1,2142 @@
+{
```

Comment:
<p>Small point "2-Dimensional" -&gt; "2-dimensional".</p><p> </p><p>Also, I genuinely don't know here, but should "Sea Level Pressure Anomalies" and "Periodogram" be capitalized?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>